aten/src/ATen/native/native_functions.yaml

```diff
@@ -5935,10 +5939,14 @@
 - func: sign_(Tensor(a!) self) -> Tensor(a!)
   use_c10_dispatcher: full
   variants: method
```

Comment:
Perhaps OT: is there a reason why `torch.sign_` does not exist? Use `variants: function, method` to make `torch.sign_` a thing.